### ISSUE TITLE
KMODE_EXCEPTION_NOT_HANDLED (1e) on systems with HVCI

### ISSUE THREAD

**Issue by nefarius** [ViGEm/ViGEmBus#7]
**Describe the bug**
On a system with Hypervisor Code Integrity (HVCI) enabled a `KMODE_EXCEPTION_NOT_HANDLED` bugcheck is triggered.

**To Reproduce**
Turn on Code Integrity checks in Driver Verifier and spawn a new x360 or ds4 pad.

**Expected behavior**
No crash obviously 😛 

**Screenshots**

**System details (please complete the following information):**
 - OS: Windows 10 (all builds)
 - Feeder software: doesn't matter
 - Driver Version: 1.14.3.0 until 1.16.102.0

**Comment by nefarius** [comment]
Output of `!wdfkd.wdflogdump ViGEmBus`:

```
57: FxIoQueue::Vf_VerifyForwardRequest - Cannot forward a request to a different WDFDEVICE 0x00007CF5A1B7A308
```

**Comment by nefarius** [comment]
[Fixed in v1.16.112](https://github.com/ViGEm/ViGEmBus/releases/tag/v1.16.112)
